test/TestRoleNames.py

```diff
@@ -0,0 +1,74 @@
+"""Test the RoleNames rule."""
+
+import pytest
+
+from ansiblelint.rules import RulesCollection
+from ansiblelint.rules.RoleNames import RoleNames
+from ansiblelint.runner import Runner
+
+RULES_COLLECTION = RulesCollection()
+RULES_COLLECTION.register(RoleNames())
```

Comment:
Why is this necessary?

Reply:
I just want to test the RoleNames rule. Should I do that differently?

Reply:
Where did you pick up this snippet? I don't recall any other tests needing register.

Reply:
Ah... I now understand why it looks weird. Other places use fixtures for test deps: https://github.com/ansible/ansible-lint/blob/0dbdbbd8196326e1c0192667ba3674f3f9f88c46/test/TestRulesCollection.py#L32.

Reply:
I refactored these lines to a fixture. Can you verify that it is the way you hinted at?

Reply:
Btw I got that register stuff from the setUp functions of the unittest test cases. Like `TestAlwaysRun`.

Reply:
Yeah, I'd rather get rid of unittest because those classes are incompatible  with pytest fixtures: https://github.com/ansible/ansible-lint/issues/725